### PR TITLE
new top level navigation

### LINKED ISSUE #367: Create "Docs and Training" menu item, landing page, and clean up menu

### PR DESCRIPTION
**Pull request by durandom**
cc @HumairAK @quaid @stefwrite @oindrillac @schwesig 

we probably want to have _some_ content in those subpages and links to previous content

fixes https://github.com/operate-first/operate-first.github.io/issues/367

### CODE REVIEW

File: config/community.yaml [hunk 1]
```diff
@@ -4,3 +4,5 @@
   href: /community/community/contributing.md
 - label: Metrics
   href: /community/community/metrics/community_metrics.ipynb
+- label: Data Sciene Community
```
Comment (oindrillac) [inline config/community.yaml]:
```suggestion
- label: Data Science Community
```

File: content/community/data-science.md [hunk 2]
```diff
@@ -10,10 +10,10 @@ We are data scientists, software engineers and DevOps professionals working with
 
 I’m a **data scientist** comfortable with Jupyter Notebooks on my laptop, but need specialized hardware and production level support to deploy my model as part of an ML application and don’t know where to start. Check out the [Espresso Series](https://www.youtube.com/playlist?list=PL8VBRDTElCWpneB4dBu4u1kHElZVWfAwW) on our YouTube channel.
 
-I’m a **software engineering manager** and I need to integrate a few data scientists and their ML models into our product, but I don’t know where to start. Take a look at our [DS Workflow](https://www.operate-first.cloud/data-science/data-science-workflows/) docs.
+I’m a **software engineering manager** and I need to integrate a few data scientists and their ML models into our product, but I don’t know where to start. Take a look at our [DS Workflow](/data-science/data-science-workflows/) docs.
```
Comment (oindrillac) [inline content/community/data-science.md]:
```suggestion
I’m a **software engineering manager** and I need to integrate a few data scientists and their ML models into our product, but I don’t know where to start. Take a look at our [DS Workflow](/data-science/ds-workflows/README.md) docs.
```

**Review by oindrillac — CHANGES_REQUESTED** [review]
@durandom changing 
content/data-science/index.md → content/community/data-science.md will break the current link https://www.operate-first.cloud/data-science. 
Can we create a redirect for www.operate-first.cloud/data-science to go to www.operate-first.cloud/community/data-science.md ? or keep the markdown where it is? so that any existing backlinks to /data-science do not break.

**Comment by oindrillac** [comment]
Existing data-science content and the [operate first data science community meetup landing page](https://www.operate-first.cloud/data-science/operate-first-data-science-community/docs/meetup-landing-page.md) are not directly navigable through the new UI. We should think about what all we want to show and how we want to restructure the documentation.

image

My proposal is to make this page ^ into a drop down and have the current "Get Involved" section under it which to show the Data Science Community Meetup landing page and also update the landing page to link the meetup within the CTAs under the "Who are you" section.
 
cc: @MichaelClifford @aakankshaduggal

**Comment by MichaelClifford** [comment]
I agree with @oindrillac I think the important navigable pages at this point are:

* https://www.operate-first.cloud/data-science/
* https://www.operate-first.cloud/data-science/operate-first-data-science-community/docs/meetup-landing-page.md
* https://www.operate-first.cloud/data-science/operate-first-data-science-community/docs/content.md

**Pull request closed without merging**